spin/cmds/pip.py

```diff
@@ -0,0 +1,38 @@
+import click
+
+from .util import run as _run
+
+
+@click.command()
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Print detailed build and installation output"
+)
+@click.argument("-e", "--editable", is_flag=True, help="Install in editable mode")
+@click.argument("pip_args", nargs=-1)
+def install(pip_args, verbose=False, editable=True, quiet=False):
+    """💽 Install package and also build.
+
+    pip arguments are passed through in the same manner as you would pip e.g.:
+
+      spin install -- -r requirements.txt
```

Comment:
Would this example make sense? I'd read it as:

```
pip install . --editable --no-build-isolation -r requirements.txt
```

which doesn't make so much sense.

Reply:
Ah okay, I clarified and edited slightly. With the new logic of the `install` function, hopefully this is better?